BenadrylAutomation/src/main/java/com/jnj/pageobjects/HomePage.java

```diff
@@ -0,0 +1,198 @@
+				"Clicking on : " + getPdct(productName).getText());
+		Action.verifyPageUrl(pdctURL);
+		Action.navigateBack();
+	}
+
+	/**
+	 * Verifies the Best Seller section by checking the displayed section, heading,
+	 * and the presence of products within the Best Seller section.
+	 *
+	 * @param expectedHeading The expected heading of the Best Seller section.
+	 */
+	public void verifyBestSeller(String expectedHeading) {
+		extentInfoLog("Bestseller Section is displayed : ",Action.isDisplayed(driver,this.bestSellerHeading));
+		String actualHeading = this.bestSellerHeading.getText();
+		Assert.assertEquals(actualHeading, expectedHeading);
+		extentInfoLog("Heading is displayed : ",actualHeading);
+		extentInfoLog("Products are displayed : ",Action.isDisplayed(driver,this.bestSellerPdct));
+	}
+
+	/**
+	 * Verifies the Related Content section by checking the displayed section, heading,
+	 * clicking on a specific content card, and verifying the resulting page URL.
+	 *
+	 * @param expectedHeading The expected heading of the Related Content section.
+	 * @param index The index of the content card to click on.
+	 * @param expectedUrl The expected URL after clicking on the content card.
+	 */
+	public void verifyRelatedContent(String expectedHeading, String index, String expectedUrl) {
+		extentInfoLog("Related Content Section is displayed : ",Action.isDisplayed(driver,this.relatedContent));
+		String actualHeading = this.relatedContent.getText();
+		Assert.assertEquals(actualHeading, expectedHeading);
+		System.out.println(this.getRelatedContentCard(index));
+		Action.performActionwithExtentInfoLog(this.getRelatedContentCard(index), "click",
+				"Clicking on : " + getRelatedContentCard(index).getText());
+		Action.verifyPageUrl(expectedUrl);
+		extentInfoLog("URL Verified!","");
+	}
+
+	/**
+	 * Verifies the Allergy Section by checking the displayed section, heading,
+	 * clicking on the "Learn More" button, and verifying the resulting page URL.
+	 * After verification, it navigates back to the previous page.
+	 *
+	 * @param sectionName The name or identifier of the Allergy Section.
+	 * @param expectedHeading The expected heading of the Allergy Section.
+	 * @param sectionBtn The identifier or information to locate the "Learn More" button.
+	 * @param expectedUrl The expected URL after clicking on the "Learn More" button.
+	 */
+	public void verifyAllergySection(String sectionName, String expectedHeading,String sectionBtn, String expectedUrl) {
+		extentInfoLog("Allergy Section is displayed : ",Action.isDisplayed(driver,this.getAllergiesSection(sectionName)));
+		String actualHeading = this.getAllergiesSection(sectionName).getText();
+		Assert.assertEquals(actualHeading, expectedHeading);
+		Action.performActionwithExtentInfoLog(this.getAllergiesLearnMore(sectionBtn), "click",
+				"Clicking on : " + getAllergiesLearnMore(sectionBtn).getText());
+		Action.verifyPageUrl(expectedUrl);
+		extentInfoLog("Successfully verified URL!","");
```

Comment:
@RashiTiwari21 Append the url verified in the log
extentInfoLog("Successfully verified URL : ",expectedUrl);

Reply:
fixed

Reply:
Verified @RashiTiwari21